README.md

```diff
@@ -68,8 +68,42 @@ module.exports = MyApiGet;
 
 All methods are optional
 
-# Get APIs with parents
+## Get APIs with parents
 
 If you have for example, a get API for a sub-entity of one specific record, the parent will be automatically be added as a filter.
 
 For example, the following endpoint: `/api/parent-entity/1/sub-entity/2`, will be a get of the sub-entity, and `parentEntity: '1'` will be set as a filter.
+
+## ✔️ Path ID validation
+Validation for path ID is available. By implementing `validateId()` method, you can make sure ID format is correct. If received ID is invalid, the API will return a ***400*** error by default and no request will be made to database.
```

Comment:
```
The `id` in the `pathParameters` can be validated if the database need it in order to avoid problems. If this feature is active, the statusCode for this kind of error will be `400`.

This validation has a default behaviour (needs Model versión .... ), and also can be customize for specific needs.
```